tests/app/main/test_suppliers.py

```diff
@@ -419,13 +389,7 @@ def test_shows_gcloud_7_in_standstill_application_passed_with_message(
             ]
         }
         data_api_client.find_frameworks.return_value = {
-            "frameworks": [
-                {
-                    'status': 'standstill',
-                    'slug': 'g-cloud-7',
-                    'name': 'G-Cloud 7'
-                }
-            ]
+            'frameworks': [self.framework(status='standstill', slug='g-cloud-7')['frameworks']]
```

Comment:
Why the use of `self.framework` from the BaseApplicationTest, which is an api_stub anyway, instead of using the stub directly?

Reply:
No real reason I suppose. `self.framework` adds the lots, which I guess made it feel more complete. But they're irrelevant here. 🤔

I'll change it to use the stub as you recommend.

Reply:
I take it back. Going to leave it as-is. The `self.framework` establishes common dates that are used across a set of tests, including this one, which gets asserted on later. Better to just have these dates in one place.